pxtblocks/blocklyimporter.ts

```diff
@@ -2,6 +2,10 @@
 /// <reference path="../built/pxtlib.d.ts" />
 
 namespace pxt.blocks {
+    export interface BlockSnippet {
+        xml: string[];
```

Comment:
why not joining the line then encoding?

Reply:
blockly's xml -> block only works on one block stack, so i have to call it separately for each top level block